### ISSUE TITLE
Shortcut don't work on MACOS

### ISSUE THREAD

**Issue by PPoooMM** [enricoros/big-AGI#437]
### Description

None of the "Chats" shortcuts work, whereas the "Edit" and "Settings" shortcuts function properly. For instance, when I attempt to open a New Chat using the "⌘ + ⌥ + N" shortcut, it does not work. 

Can this be fixed? It would be highly beneficial for executing tasks more efficiently.

**Chats**
⌘ + ⌥ + Left | Previous chat (in history)
-- | --
⌘ + ⌥ + Right | Next chat (in history)
⌘ + ⌥ + N | New chat
⌘ + ⌥ + X | Reset chat
⌘ + ⌥ + D | Delete chat
⌘ + ⌥ + B | Branch chat

Thank you.

### Device and browser

Macbook M1 Pro, 
OS Mac Sonoma 14.3.1,
Microsoft Edge

### Screenshots and more

![CleanShot 2567-02-29 at 11 25 20@2x](https://github.com/enricoros/big-AGI/assets/123398360/d7099517-d3e0-4c92-a019-c6b0746f59ef)

### Willingness to Contribute

- [ ] 🙋‍♂️ Yes, I would like to contribute a fix.

**Comment by enricoros** [comment]
You're right, those are absolutely beneficial. I don't have a MacBook - would need someone that has a Mac to debug what's not working with:
https://github.com/enricoros/big-AGI/blob/70474ce517df82aab32abc20f314ae424977104e/src/common/components/useGlobalShortcut.ts#L24

**Comment by PPoooMM** [comment]
> You're right, those are absolutely beneficial. I don't have a MacBook - would need someone that has a Mac to debug what's not working with:
> 
> https://github.com/enricoros/big-AGI/blob/70474ce517df82aab32abc20f314ae424977104e/src/common/components/useGlobalShortcut.ts#L24

I don't know much about programming languages, but I assume that combining '⌘ + ⌥' doesn't work on macOS.  However, '⌘ + ⇧', '⇧ + Any Key', or '⌥ + Enter' work perfectly. 

Can you change a default shortcut on macOS that does not begin with '⌘ + ⌥'? 
For example, changing from '⌘ + ⌥ + N' for New Chat to '⌘ + ⇧ + [' instead, or creating a custom shortcut for one's own configuration. 

Additionally, these shortcuts are not the same as the default shortcuts on macOS.

**Comment by enricoros** [comment]
Thanks, we made the UI to be clean. Shortcuts hell a lot also.

Do you think that shortcuts should be Command + Control + any, where today we have Command + Option + any? I could implement that.

**Comment by PPoooMM** [comment]
> Thanks, we made the UI to be clean. Shortcuts hell a lot also.
> 
> Do you think that shortcuts should be Command + Control + any, where today we have Command + Option + any? I could implement that.

I think it works (it may be good to try) because in Microsoft Edge, many extensions can add custom keyboard shortcuts (see my uploaded picture for an example; you can visit this URL: edge://extensions/shortcuts). I can't create a shortcut using '⌘ + ⌥ + Any key', but I can use '⌘ + ⌃ + Any key' (Command + Control + Any key). Is it worth trying?

![CleanShot 2567-03-02 at 10 19 59@2x](https://github.com/enricoros/big-AGI/assets/123398360/9c04e668-24bf-4560-b1b8-227204709e82)

**Comment by enricoros** [comment]
I've made some progress today, but still don't have a system to verify. I see what you mean by having those shortcuts unavailable. As soon as I get a macbook I'll work on this.

**Comment by enricoros** [comment]
Closing as obsolete.

The Cmd+Option scheme you hit no longer exists. Shortcuts are all Ctrl(+Shift)+key now, which is Control on macOS, and the shortcuts modal renders it as the Control glyph. New chat is Ctrl+Shift+N. The command key is deliberately unused because it collides with browser shortcuts.

I said I would get a MacBook and fix this and never did - what actually fixed it was the scheme changing. Your other ask, setting your own combos, is tracked in #895 and now has an approved direction. Closing this one.

@PPoooMM this close comes from an automated backlog pass - if you disagree, feel free to reopen.
